### ISSUE TITLE
TypeError: Cannot read property 'lang' of undefined

### ISSUE THREAD

**Issue by shadrachtuck** [Leecason/element-tiptap#144]
![Screen Shot 2020-07-09 at 3 11 30 PM](https://user-images.githubusercontent.com/33300343/87091264-a0ac8d80-c1f6-11ea-9dd0-7ac72be64d9b.png)
I'm getting this error every time I try to boot up my project and it's breaking my code. I believe it has something to do with i18 but am not positive. Can try to replicate in code sandbox.

**Comment by shadrachtuck** [comment]
Yes, I can confirm that once I rolled back from the current version to version 1.24.4 before you implemented the i18 Russian language addition, the error disappears and my project compiles fine.

**Comment by Leecason** [comment]
Try remove `node_modules` and lock files then reload packages

**Comment by shadrachtuck** [comment]
Started afresh with the packages after downgrading and the same error re-appeared. Then I went back to 1.24.5 where you added the ru i18 package and it worked fine. Not sure which release produces the error but the most recent release doesn't work for me. I wonder if anyone else is encountering this issue or if it's because I'm running it within electron.

**Comment by jcc-alfred** [comment]
这个问题解决了么？

**Comment by zhaozwgithub** [comment]
我使用@1.26.2版本时，全局注册不会出现“TypeError: Cannot read property 'lang' of undefined”这个错误。
但是局部注册会出现这个错误“TypeError: Cannot read property 'lang' of undefined”
我在使用
          <el-tiptap
            v-model="article.content"
            :extensions="extensions"
            height="350"
            lang="zh"
            placeholder="请输入文章内容"
          ></el-tiptap>
加入lang="zh"之后错误消失。

**Comment by zhaozwgithub** [comment]
我使用@1.26.2版本时，全局注册不会出现“TypeError: Cannot read property 'lang' of undefined”这个错误。
但是局部注册会出现这个错误“TypeError: Cannot read property 'lang' of undefined”
我在使用
![Uploading image.png…]()
加入lang="zh"之后错误消失。

**Comment by zhaozwgithub** [comment]
我在使用@1.26.2版本时，全局注册不会出现“TypeError: Cannot read property 'lang' of undefined”这个错误。
但是局部注册会出现这个错误“TypeError: Cannot read property 'lang' of undefined”
我在使用
`<el-tiptap
            v-model="article.content"
            :extensions="extensions"
            height="350"
            lang="zh"
            placeholder="请输入文章内容"
          ></el-tiptap>`
加入lang="zh"之后错误消失。

**Comment by jie123js** [comment]
我也遇到这个问题  局部注册 楼上说的对

**Comment by Chrisimir** [comment]
Can confirm that adding `:lang="this.$i18n.locale"` solves this issue.
Or `lang="en"` if you're not using i18n.

This works
`<el-tiptap
                v-model="wysiwygContent"
                :extensions="extensions"
                :lang="this.$i18n.locale"
                output="html"
                @onUpdate="onEditorUpdate"
            />`

Thanks @zhaozwgithub
